{{cookiecutter.project_slug}}/clients/web/react/src/utils/server-errors.ts

```diff
@@ -0,0 +1,23 @@
+import { isAxiosError, AxiosError } from 'axios'
+
+export function handleDRFAPIErrorsOrReThrow(error: AxiosError | Error): string[]{
```

Comment:
I'd say error is `unknown`

Reply:
`AxiosError` type is inferred by the `isAxiosError` assert function on it's own, no need to declare it at all